copilot/fastapi_k8s_3.py

```
@@ -0,0 +1,47 @@
+
+app = FastAPI()
+
+def get_k8s_client():
+    config.load_kube_config()
+    return client.ApiClient()
+
+@app.api_route("/argocd-passthru/{path:path}", methods=["GET", "POST", "PUT", "DELETE", "PATCH"])
+async def argocd_passthru(
+    path: str,
+    request: Request,
+    k8s_client: client.ApiClient = Depends(get_k8s_client)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)